### DISCUSSION TITLE
What is the Basic Paradigm of Writing Backpressure

### DISCUSSION THREAD

**Discussion by haoynmail** [uNetworking/uWebSockets.js#942] [Q&A]
The example Backpressure.js says "The following is a (preposterous) example of how data can be pushed according to backpressure.
Do not take this as a way to actually write code." , so is it the correct way ?

**Comment by uNetworkingAB** [comment]
Ah, I can see the confusion. But that's actually a good thing. It makes it a reader's task to figure out.

**Reply by haoynmail** [reply]
Thank you for your reply,

I don't understand what you mean. Do you mean that the Backpressure.js example is the correct way to handle back pressure

**Reply by haoynmail** [reply]
I want to write a backpressure processing program. Can I write it in the Backpressure.js mode?

**Reply by e3dio** [reply]
You can write your code however you want, the Backpressure.js example keeps backpressure full at a certain level, you don't have to do this but you can

**Reply by haoynmail** [reply]
thank you
